packages/plugins/newrelic/src/index.ts

```diff
@@ -1,8 +1,7 @@
 import { shim as instrumentationApi } from 'newrelic';
-import { Plugin, OnResolverCalledHook } from '@envelop/types';
+import { Plugin, OnResolverCalledHook, isAsyncIterable } from '@envelop/types';
```

Comment:
exporting this from `@envelop/types` makes it no longer a type-only package 🤔  Should we move it to the core instead?

Reply:
I wanted to put that in the core, but then it means that the plugins know the core, which might not be ideal? I know the name `types` is confusing in that context, but it's more like a `common` is `shared` now